demisto_sdk/commands/common/update_id_set.py

```diff
@@ -803,12 +890,13 @@ def re_create_id_set(id_set_path: str = "./Tests/id_set.json", objects_to_create
     layouts_list = []
     reports_list = []
     widgets_list = []
+    mappers_list = []
 
     pool = Pool(processes=cpu_count() * 2)
 
     print_color("Starting the creation of the id_set", LOG_COLORS.GREEN)
 
-    with click.progressbar(length=12, label="Progress of id set creation") as progress_bar:
+    with click.progressbar(length=13, label="Progress of id set creation") as progress_bar:
```

Comment:
```suggestion
    with click.progressbar(length=len(objects_to_create), label="Progress of id set creation") as progress_bar:
```
i think it might make it easier for the future